### ISSUE TITLE
Run on PRs

### ISSUE THREAD

**Issue by jacobtomlinson** [dependencyci/support#99]
It would be great if Dependency CI could run on PRs as well as the master branch. I'm only discovering dependancy issues once they've been merged to master in opsdroid/opsdroid.

**Comment by katzj** [comment]
If you look on the repo list on the right side, there is a button that should say `Enable CI` that will let you turn on checking of pull requests. If you don't see it, you may have to ask an admin for your github org to add dependencyci to have access (https://github.com/settings/connections/applications/89fef37df2df5a4ae166)

**Comment by jacobtomlinson** [comment]
Yep I have enabled CI but it doesn't seem to check PRs, only the master
branch.

**Comment by katzj** [comment]
I just tried force-re-enabling the webhook. If that didn't work, can you provide a screenshot for https://github.com/opsdroid/opsdroid/settings/hooks? You should see hooks.dependencyci.com in the list like the below

<img width="867" alt="screen shot 2018-02-12 at 10 21 49 am" src="https://user-images.githubusercontent.com/141657/36103904-9489c60c-0fde-11e8-90a7-dff7ed496683.png">
